tests/test_version.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf8 -*-
+
+# TODO: This is a demo of the way we can report he cluster version.
+# Code here will be included in some python module when the current changes
+# settles down. Also for this report to be actually useful, we need to run this
+# as autouse fixture (again, exact way to do this needs to be aligned with
+# others, eg. https://github.com/red-hat-storage/ocs-ci/pull/173).
+
+
+from datetime import datetime
+import argparse
+import logging
+import os.path
+import sys
+
+from ocs.ocp import OCP
+import ocsci.config
+
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: convert this to a fixture later, align with setup, teardown and testing
+# @pytest.fixture(scope="session", autouse=True)
+def test_version():
```

Comment:
really good helper function.
I think it will be great to call it from as a first "test" when running tests - like in a setup of a session